
 Upstream patch, backported from 4.83A-linux.
                          -- kzak 27-Jul-2009

diff -up lsof_4.82-rh/Configure.kzak lsof_4.82-rh/Configure
--- lsof_4.82-rh/Configure.kzak	2009-07-27 09:50:15.000000000 +0200
+++ lsof_4.82-rh/Configure	2009-07-27 09:50:43.000000000 +0200
@@ -2700,7 +2700,7 @@ return(0); }
 	LSOF_TMP1=1
       fi	# }
     else
-      if test "X$LINUX_HASSELINUX" = "XY" -o "X$LINUX_HASSELINUX" = "xY" # {
+      if test "X$LINUX_HASSELINUX" = "XY" -o "X$LINUX_HASSELINUX" = "Xy" # {
       then
 	LSOF_TMP1=1
       fi	# }
